inst/@sym/inv.m

```diff
@@ -51,30 +51,21 @@
 
 function z = inv(x)
 
-  [n,m] = size(x);
-  if n ~= m
-    error('matrix is not square')
-  end
-
```

Comment:
I like this change.

But can use also add a test that we get an error if you try to invert a non-square matrix (my fault: I should've added this originally)